common/src/main/java/com/wynntils/screens/maps/PoiManagementScreen.java

```diff
@@ -568,7 +568,7 @@ public void deletePoi(CustomPoi poiToDelete) {
 
         customPois.get().remove(poiToDelete);
         customPois.touched();
-        Managers.Feature.getFeatureInstance(MainMapFeature.class).updateWaypoints();
+        // Managers.Feature.getFeatureInstance(MainMapFeature.class).updateWaypoints();
```

Comment:
Why is this commented out? If it is not needed anymore, remove it?

Reply:
The PoiManagmentScreen needs full porting, I've not done anything in that regard in this PR. I did not want to remove these old update methods, for anyone porting in the future, it might be useful. This'll be removed before the actual mapdata PR is merged.